src/chatd.cpp

```diff
@@ -3847,7 +3847,7 @@ bool Chat::findLastTextMsg()
         if (wptr.deleted())
             return;
 
-        if (mOnlineState == kChatStateOnline)
+        if (mOnlineState == kChatStateOnline && !mLastTextMsg.isFetching())
```

Comment:
I don't know if it would be more correct using `mServerFetchState` in stead of !mLastTextMsg.isFetching() or maybe set the condition inside of `requestHistoryFromServer`, in this way, we avoid send two `HIST` commands in all cases